extension/js/common/platform/store/contact-store.ts

```diff
@@ -368,7 +368,14 @@ export class ContactStore extends AbstractStore {
     const name = emailEntity.name ? emailEntity.name.toLowerCase() : '';
     // we only need the longest word if it starts with a shorter one,
     // e.g. we don't need "flowcrypt" if we have "flowcryptcompatibility"
-    const sortedNormalized = [...Str.splitAlphanumericExtended(email), ...Str.splitAlphanumericExtended(name)].filter(p => !!p)
+    // also, filter out top level domains from emails
+    const emailTokens = Str.splitAlphanumericExtended(email);
+    // handle correctly email like user@host
```

Comment:
how is it possible to get an email address like `user@host`?

Reply:
I don't know, just potentially possible. Maybe in some test. But ok, will rewrite code.